apps/test-viewer/src/components/ViewerOptions.tsx

```diff
@@ -0,0 +1,52 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { StatusBarItem, StatusBarSection, UiItemsProvider } from "@itwin/appui-react";
+import { SvgVisibilityShow } from "@itwin/itwinui-icons-react";
+import { createContext, PropsWithChildren, useContext, useState } from "react";
+
+export interface ViewerOptionsContext {
```

Comment:
Maybe we can split this context into `ViewerOptionsContext` and `ViewerActionsContext`? I think that would be a little bit cleaner for components that only used options values and does not modify them.